tests/unittests/video_tests.py

```
@@ -164,13 +181,61 @@ def test_video_indexes(self):
         dataset.drop_index("frames.cls.label")
         self.assertNotIn("frames.cls.label", dataset.list_indexes())
 
-        compound_index_name = dataset.create_index(
-            [("frames.id", 1), ("frames.field", 1)]
+        dataset.create_index([("frames.field", -1)])
+        indexes = dataset.get_index_information()
+        indexes["frames.field"]["key"] == [("field", -1)]
```

Ruff (0.13.1)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)